### ISSUE TITLE
Add input controllers support

### ISSUE THREAD

**Issue by scemino** [scemino/engge#39]
Support for game controller, touch pads, etc.

**Comment by Krish2882005** [comment]
Still Need To Add Input Support For It?

**Comment by scemino** [comment]
> Still Need To Add Input Support For It?

Yes, but these days, I'm integrating my project Engge Framework in engge, and then it should be easier to fix this issue. Engge Framework is based on SDL2.

**Comment by Krish2882005** [comment]
I Know SDL2
